src/api/project/project.controller.ts

```diff
@@ -12,4 +12,13 @@ export default class ProjectController {
       next(new HttpException(error.statusCode || 500, error.message));
     }
   }
+
+  async show(req: Request, res: Response, next: NextFunction) {
+    try {
+      const result = await projectService.getAllProject();
+      res.send(result);
+    } catch (error) {
+      next(new HttpException(error.statusCode || 500, error.message));
+    }
+  }
```

Comment:
kalau untuk nampilin semua data biasanya methodnya index sih
```suggestion
  async index(req: Request, res: Response, next: NextFunction) {
    try {
      const result = await projectService.getAllProject();
      res.send(result);
    } catch (error) {
      next(new HttpException(error.statusCode || 500, error.message));
    }
  }
```

Reply:
iya biasanya show itu buat nampilin 1 data aja